src/server.rs

```diff
@@ -430,11 +430,12 @@ impl ResolvesServerCert for ResolvesServerCertFromChoices {
 }
 
 /// The TLS Client Hello information provided to a ClientHelloCallback function.
-/// `sni_name` is the SNI servername provided by the client. If the client
-/// did not provide an SNI, the length of this `rustls_string` will be 0. The
-/// signature_schemes carries the values supplied by the client or, should
-/// the client not use this TLS extension, the default schemes in the rustls
-/// library. See: <https://docs.rs/rustls/0.20.0/rustls/internal/msgs/enums/enum.SignatureScheme.html>.
+/// `server_name` is the value of the ServerNameIndication extension provided
+/// by the client. If the client did not send an SNI, the length of this
+/// `rustls_string` will be 0. The signature_schemes field carries the values
+/// supplied by the client or, if the client did not send this TLS extension,
+/// the default schemes in the rustls library. See:
+/// <https://docs.rs/rustls/0.20.0/rustls/internal/msgs/enums/enum.SignatureScheme.html>.
```

Comment:
Are you leaving these doc ref's at 0.20.0 or should they be updated? I was updating some I came across in https://github.com/rustls/rustls-ffi/pull/303/files

Reply:
I'm leaving them for now and figure we'll do one big batch of updates. Speaking of which, I'll file an issue for that and add it to the tracker.

Reply:
SGTM. I can revert that part of my diff when I pick it up again tomorrow!